[2024-06-05T15:14:11.210+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T15:14:11.226+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: metrics_calculator.statistics_calculator manual__2024-06-05T15:14:06.542614+00:00 [queued]>
[2024-06-05T15:14:11.230+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: metrics_calculator.statistics_calculator manual__2024-06-05T15:14:06.542614+00:00 [queued]>
[2024-06-05T15:14:11.230+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-05T15:14:11.237+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): statistics_calculator> on 2024-06-05 15:14:06.542614+00:00
[2024-06-05T15:14:11.242+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=130) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-05T15:14:11.243+0000] {standard_task_runner.py:63} INFO - Started process 132 to run task
[2024-06-05T15:14:11.244+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'metrics_calculator', 'statistics_calculator', 'manual__2024-06-05T15:14:06.542614+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/dag_reader.py', '--cfg-path', '/tmp/tmpcqipz68e']
[2024-06-05T15:14:11.245+0000] {standard_task_runner.py:91} INFO - Job 50: Subtask statistics_calculator
[2024-06-05T15:14:11.272+0000] {task_command.py:426} INFO - Running <TaskInstance: metrics_calculator.statistics_calculator manual__2024-06-05T15:14:06.542614+00:00 [running]> on host 9d270a478535
[2024-06-05T15:14:11.321+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mehdizarria' AIRFLOW_CTX_DAG_ID='metrics_calculator' AIRFLOW_CTX_TASK_ID='statistics_calculator' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T15:14:06.542614+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-05T15:14:06.542614+00:00'
[2024-06-05T15:14:11.322+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T15:14:15.375+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-05T15:14:15.380+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T15:14:15.422+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=metrics_calculator, task_id=statistics_calculator, run_id=manual__2024-06-05T15:14:06.542614+00:00, execution_date=20240605T151406, start_date=20240605T151411, end_date=20240605T151415
[2024-06-05T15:14:15.459+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T15:14:15.492+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-05T15:14:15.494+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
